units/systemd-udevd.service.in

```diff
@@ -8,7 +8,7 @@
 #  (at your option) any later version.
 
 [Unit]
-Description=udev Kernel Device Manager
+Description=Rule-based manager for device events and files
```

Comment:
we so far uppercase most words in the description strings of all our services

Reply:
I'll submit a fix.